io.github.MakovWait.Godots.json

```
@@ -0,0 +1,108 @@
+{
+    "id" : "io.github.MakovWait.Godots",
+    "runtime" : "org.freedesktop.Sdk",
+    "runtime-version" : "23.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11",
+        "org.freedesktop.Sdk.Extension.dotnet7"
+    ],
+    "command": "godots",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet7/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet7/lib",
+        "env": {
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig"
+        }
+    },
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=x11",
```

```suggestion
        "--socket=x11",
```

fallback-x11 is for when wayland isn't available, but you didn't indicate wayland support. it's either or anyway.

---

:+1: I'll correct it then.